src/neo-vm/ExecutionEngine.cs

```diff
@@ -308,7 +312,23 @@ private bool ExecuteInstruction()
                                 return false;
                             break;
                         }
-
+                    // Static Stack ops
+                    case OpCode.DUPFROMSTATICSTACK:
+                        {
+                            context.EvaluationStack.Push(this.StaticEvaluationStack.Peek());
+                            if (!CheckStackSize(true)) return false;
+                            break;
+                        }
+                    case OpCode.TOSTATICSTACK:
+                        {
+                            context.AltStack.Push(this.StaticEvaluationStack.Pop());
```

Comment:
This should be:
context.StaticStack.Push ( EvaluationStack.pop )

Reply:
right